app/layout/nav_menu.phtml

```diff
@@ -131,20 +131,38 @@
 
 	<?php $url_output = Minz_Request::currentRequest(); ?>
 	<div class="stick" id="nav_menu_views">
-		<?php $url_output['a'] = 'normal'; ?>
-		<a class="view_normal btn <?php echo $actual_view == 'normal'? 'active' : ''; ?>" title="<?php echo _t('index.menu.normal_view'); ?>" href="<?php echo Minz_Url::display($url_output); ?>">
-			<?php echo _i("view-normal"); ?>
-		</a>
+		<?php
+		$readingModes = array(
+			'normal' => array(
+				'name' => _i("view-normal"),
+				'title' => _t('index.menu.normal_view'),
+				'url' => array_merge($url_output, array('c' => Minz_Request::defaultControllerName(), 'a' => 'normal')),
+				'active' => (Minz_Request::controllerName() == Minz_Request::defaultControllerName() && $actual_view == 'normal'),
```

Comment:
Please put the "fast" string comparisons first, and use triple-equal, like:

```php
'active' => $actual_view === 'normal' && Minz_Request::controllerName() === Minz_Request::defaultControllerName(),
```

and maybe the multiple `Minz_Request::controllerName() === Minz_Request::defaultControllerName()` could be factorised somehow